doc/source/analyzing/generating_processed_data.rst

```diff
@@ -9,6 +9,42 @@ data by hand and construct plots which can then be combined with other plots,
 modified in some way, or even (gasp) created and modified in some other tool or
 program.
 
+.. _exporting-container-data:
+
+Exporting Container Data
+------------------------
+
+Fields from data containers such as regions, spheres, cylinders, etc. can be exported
+tabular format using either a :class:`~pandas.DataFrame` or an :class:`~astropy.table.QTable`. 
+
+To export to a :class:`~pandas.DataFrame`, use 
+:meth:`~yt.data_objects.data_containers.YTDataContainer.to_dataframe`:
+
+.. code-block:: python
+
+    sp = ds.sphere("c", (0.2, "unitary"))
```

Comment:
not necessary for this PR, but maybe we want to issue a warning message if no fields have been loaded into memory? That way somebody doesn't generate an empty dataframe (or astropy table)?